Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -2284,11 +2284,11 @@ - (void)optionalFormButtonTapped:(id)sender {
     
     // Test expected Single Choice
     {
-        ORKAnswerFormat *answer = [ORKAnswerFormat choiceAnswerFormatWithStyle:ORKChoiceAnswerStyleSingleChoice textChoices:[NSArray arrayWithObjects:@"Choose Yes",@"Choose No", nil]];
+        ORKAnswerFormat *answer = [ORKAnswerFormat choiceAnswerFormatWithStyle:ORKChoiceAnswerStyleSingleChoice textChoices:[NSArray arrayWithObjects:@"Choose Yes", @"Choose No", nil]];
         ORKQuestionStep *step = [ORKQuestionStep questionStepWithIdentifier:@"question_02"
                                                                       title:@"This is a single-choice question."
                                                                      answer:answer];
-        step.optional = NO;
+        step.optional = YES;
```

Comment:
Since `optional` defaults to `YES`, how about removing all the optional lines? ;-)